tardis/plasma/properties/property_collections.py

```diff
@@ -1,5 +1,15 @@
+from tardis.plasma.properties import (BetaRadiation, LevelBoltzmannFactor,
+    Levels, Lines, SelectedAtoms, AtomicMass, LTEPartitionFunction,
+    LevelPopulationLTE, LevelNumberDensity, PhiSahaLTE, GElectron,
+    IonizationData, NumberDensity, IonNumberDensity, LinesLowerLevelIndex,
+    LinesUpperLevelIndex, TauSobolev, TRadiative, AtomicData, Abundance,
+    Density, TimeExplosion)
 
 class LTEInputs(list):
-    pass
-
-
+    def __init__(self):
```

Comment:
@wkerzendorf Yeah, I think the property groups make a lot of sense! It makes things a lot tidier anyway.